runtime/jvmti/jvmtiThread.c

```diff
@@ -408,6 +407,8 @@ jvmtiInterruptThread(jvmtiEnv *env,
 		
 		ENSURE_PHASE_LIVE(env);
 		ENSURE_CAPABILITY(env, can_signal_thread);
+		ENSURE_JTHREAD_NON_NULL(thread);
+		ENSURE_JTHREAD(currentThread, thread);
```

Comment:
If possible, it would be nice to combine this, as the original `ENSURE_JTHREAD_NON_NULL` is completely misleading (it does not check for thread at all, just the NULL). Alternately, replace all uses of `ENSURE_JTHREAD_NON_NULL` with `ENSURE_JOBJECT_NON_NULL` and delete the misleading one entirely.